byron/ledger/formal-spec/properties.tex

```diff
@@ -156,3 +184,31 @@ \subsection{UTxO Properties}
   $$
 
 \end{property}
+
+Property~\ref{prop:utxo-money-supply-cnst} models the fact that the amount of
+money in the system (counted as Lovelaces) remains constant.
+
+\begin{property}[Money supply is constant in the system]\label{prop:utxo-money-supply-cnst}
+  For all
+
+  $$
+  \left(
+    \begin{array}{l}
+      \var{utxo_0}\\
+      \var{reserves_0}
+    \end{array}
+  \right)
+  \transtar{\hyperref[fig:rules:utxo]{utxo}}{\txs}
+  \left(
+    \begin{array}{l}
+      \var{utxo}\\
+      \var{reserves}
+    \end{array}
+  \right)
+  $$
+
+  we have:
+
+  $$ \var{reserves} + \balance{utxo} =  \var{reserves_0} - \balance{utxo_0} $$
```

Comment:
Why is this `reserves_0 - utxo_0`, rather than `reserves_0 + utxo_0`? Are we defining `reserves_0` as `lovelaceCap`?

Reply:
Looks like you defined `reserves_0` as `lovelaceCap - balance (utxo_0)` further down, so this should probably be a `+`

Reply:
Good catch Ru, I missed that when re-writing the property :pray: